generator/transformers/common_producer.py

```diff
@@ -168,7 +168,7 @@ def nullable(type_native: str, mandatory: bool) -> str:
         :param mandatory: is parameter mandatory
         :return: string with modificator
         """
-        if mandatory or re.match(r'BOOL|float|double', type_native):
+        if mandatory or re.match(r'BOOL|double', type_native):
```

Comment:
I think we can also get rid of the `double` since it appears we only have `SDLFloat` in **NSNumber+NumberType.h**.